src/components/pages/admin.yml

```diff
@@ -4,3 +4,6 @@ settingsTabTitle:
 storagePlansTabTitle:
   en: Storage plans
   de: Speicherpläne
+testsTabTitle:
+  en: Tests
+  de: Teste
```

Comment:
"Teste" is only used for medical tests, otherwise the plural is also "Tests" in German.

Reply:
I see 🤔 Corrected!